cli/src/args.rs

```diff
@@ -158,6 +162,18 @@ pub(crate) enum CliDownloadSequence {
     Ask,
 }
 
+/// Whether to abort the download when a case in a sequence is not found.
+#[derive(Debug, Clone, Serialize, Default, ValueEnum)]
+pub(crate) enum CliSequenceErrorHandling {
```

Comment:
Same as for SequenceErrorHandling (see below), this can probably also derive `Copy`

Reply:
Yes, done now.